docs/project/view.md

```diff
@@ -0,0 +1,280 @@
+{% swagger-description %}  
+  
+{% endswagger-description %}  
+{% swagger-response status="external_url_len ( felt )" description="" %}  
+{% endswagger-response %}  
+{% swagger-response status="external_url ( felt* )" description="" %}  
+{% endswagger-response %}  
+{% endswagger %}  
+{% swagger method = "view" path = " " baseUrl = " " summary = "description" %}  
+{% swagger-description %}  
+  
+{% endswagger-description %}  
+{% swagger-response status="description_len ( felt )" description="" %}  
+{% endswagger-response %}  
+{% swagger-response status="description ( felt* )" description="" %}  
+{% endswagger-response %}  
+{% endswagger %}  
+{% swagger method = "view" path = " " baseUrl = " " summary = "holder" %}  
+{% swagger-description %}  
+  
+{% endswagger-description %}  
+{% swagger-response status="holder_len ( felt )" description="" %}  
+{% endswagger-response %}  
+{% swagger-response status="holder ( felt* )" description="" %}  
+{% endswagger-response %}  
+{% endswagger %}  
+{% swagger method = "view" path = " " baseUrl = " " summary = "certifier" %}  
+{% swagger-description %}  
+  
+{% endswagger-description %}  
+{% swagger-response status="certifier_len ( felt )" description="" %}  
+{% endswagger-response %}  
+{% swagger-response status="certifier ( felt* )" description="" %}  
+{% endswagger-response %}  
+{% endswagger %}  
+{% swagger method = "view" path = " " baseUrl = " " summary = "land" %}  
+{% swagger-description %}  
+  
+{% endswagger-description %}  
+{% swagger-response status="land_len ( felt )" description="" %}  
+{% endswagger-response %}  
+{% swagger-response status="land ( felt* )" description="" %}  
+{% endswagger-response %}  
+{% endswagger %}  
+{% swagger method = "view" path = " " baseUrl = " " summary = "unit_land_surface" %}  
+{% swagger-description %}  
+  
+{% endswagger-description %}  
+{% swagger-response status="unit_land_surface_len ( felt )" description="" %}  
+{% endswagger-response %}  
+{% swagger-response status="unit_land_surface ( felt* )" description="" %}  
+{% endswagger-response %}  
+{% endswagger %}  
+{% swagger method = "view" path = " " baseUrl = " " summary = "country" %}  
+{% swagger-description %}  
+  
+{% endswagger-description %}  
+{% swagger-response status="country_len ( felt )" description="" %}  
+{% endswagger-response %}  
+{% swagger-response status="country ( felt* )" description="" %}  
+{% endswagger-response %}  
+{% endswagger %}  
+{% swagger method = "view" path = " " baseUrl = " " summary = "expiration" %}  
+{% swagger-description %}  
+  
+{% endswagger-description %}  
+{% swagger-response status="expiration_len ( felt )" description="" %}  
+{% endswagger-response %}  
+{% swagger-response status="expiration ( felt* )" description="" %}  
+{% endswagger-response %}  
+{% endswagger %}  
+{% swagger method = "view" path = " " baseUrl = " " summary = "total_co2_sequestration" %}  
```

Comment:
Some of the parameters, like this one, have no description of comments.

Do we need to add them some comment or remove them from Gitbook ?

Reply:
All pseudo-metadata will be removed after the refactoring sprint in favor of ipfs mechanism.
That's why no doc has been added on them.